data_folder/config.yaml

```diff
@@ -39,4 +39,8 @@ companyBlacklist:
 
 titleBlacklist:
   - word1
-  - word2
\ No newline at end of file
+  - word2
+
+llm_model_type: [openai / ollama]
+llm_model: ['gpt-4o' / 'mistral:v0.3']
+llm_api_url: [https://api.pawan.krd/cosmosrp/v1',  http://127.0.0.1:11434/]
```

Comment:
this yaml doesn't look right to me

Reply:
This is for showing all the options - corrected the yaml file